synapse/app/admin_cmd.py

```diff
@@ -65,16 +68,11 @@ class AdminCmdSlavedStore(
 
 
 class AdminCmdServer(HomeServer):
-    DATASTORE_CLASS = AdminCmdSlavedStore
+    DATASTORE_CLASS = AdminCmdSlavedStore  # type: ignore
```

Comment:
I'm essentially punting handling this attribute (for the 3 sub-classes of `HomeServer`) to #11165.